frameworks/cassandra/README.md

```diff
@@ -404,15 +404,15 @@ The DC/OS Apache Cassandra Service currently exposes only the `node` endpoint ty
     "10.0.1.27:9042"
   ],
   "dns": [
-    "node-2-server.cassandra.mesos:9042",
-    "node-0-server.cassandra.mesos:9042",
-    "node-1-server.cassandra.mesos:9042"
+    "node-2-server.cassandra.autoip.dcos.thisdcos.directory:9042",
```

Comment:
I know you covered this, but remind me: this address resolves correctly whether or not the task is a member of an overlay network, correct?